packages/roosterjs-editor-dom/lib/index.ts

```diff
@@ -50,6 +50,7 @@ export { default as setColor } from './utils/setColor';
 export { default as matchesSelector } from './utils/matchesSelector';
 export { default as createElement, KnownCreateElementData } from './utils/createElement';
 export { default as moveChildNodes } from './utils/moveChildNodes';
+export { saveTableInfo, getTableFormatInfo } from './utils/tableFormatInfo';
```

Comment:
Suggest do not export these 2 functions from index.ts. We don't expect customer app call these 2 functions directly.

For demo site, you can do deep import to that file. It is ok to do some kind of hack in demo site since it is not production code